tests/Feature/GroupBalancedDivisionTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+use App\Helpers\GroupBalancedDivision;
+use App\Models\Event;
+use App\Models\User;
+use App\Models\Course;
+use App\Models\Registration;
+use App\Models\Group;
+
+
+uses(\Illuminate\Foundation\Testing\RefreshDatabase::class);
+
+test('equal div only drinkers', function () {
+    $event = Event::factory()->create(['consider_alcohol' => true])->first();
```

Comment:
Code is repeated redunantly here. Use the hooks from Pest such as `beforeEach`. See also: https://pestphp.com/docs/hooks

Reply:
done